packages/teleport/src/Support/ScheduleUpgrades/ScheduleUpgrades.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { ButtonPrimary, ButtonSecondary } from 'design/Button';
+import Dialog, {
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from 'design/Dialog';
+import Select, { Option } from 'shared/components/Select';
+import { Attempt } from 'shared/hooks/useAttemptNext';
+import { UpgradeWindowStart } from 'teleport/services/upgradeWindow/upgradeWindow';
+import Text from 'design/Text';
+import Alert from 'design/Alert';
```

Comment:
i haven't been paying attention to these, but just like how teleport has import ordering, webapps has its own: https://gravitational.slack.com/archives/C02DQ1C2BMW/p1643999171617869

Reply:
I'm becoming less okay with us just linking to a random slack thread from 5 months ago and expecting that people should have just known it existed.

Import order is largely a matter of personal preference and doesn't affect functionality in any way.

For things like this (no impact on behavior and hard to get everyone to agree on) we need to decide whether or not we care.

- If we care: we need to enforce it with tooling and not tribal knowledge so that devs don't ever have to think about it.
- Otherwise: we don't care and devs should simply not have to think about it.